gs-element-blockly.html

```diff
@@ -594,13 +594,15 @@ <h2>Hello gs-element-blockly</h2>
             if (icon) {
               this.appendDummyInput().appendField(new Blockly.FieldImage(
                 icon,
-                16,
-                16,
+                Blockly.currentDisplayMode().iconSize,
+                Blockly.currentDisplayMode().iconSize,
                 finalName
               ));
             }
 
-            self._definePrimitiveHolesOrDropdown(this, parts, dropdownOptions);
+            if (Blockly.currentDisplayMode().showText) {
```

Comment:
Perhaps it would be nice to make display mode polymorphic instead of using type tests.

Reply:
Lo pensé, pero habría que mover al modo las definiciones de los bloques y me pareció que por ahora no se justificaba la indirección.